drivers/sensor/bmm150/bmm150.c

```diff
@@ -0,0 +1,602 @@
+/* bmm150.c - Driver for Bosch BMM150 Geomagnetic Sensor */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bmm150.h"
+
+static const struct {
+	int freq;
+	u8_t reg_val;
+} bmm150_samp_freq_table[] = { {2, 0x01},
+			       {6, 0x02},
+			       {8, 0x03},
+			       {10, 0x00},
+			       {15, 0x04},
+			       {20, 0x05},
+			       {25, 0x06},
+			       {30, 0x07} 
+};
+
+static const struct bmm150_preset {
+	u8_t rep_xy;
+	u8_t rep_z;
+	u8_t odr;
+} bmm150_presets_table[] = {
+	[LOW_POWER_PRESET] = {3 , 3, 10},
+	[REGULAR_PRESET] = {9, 15, 10},
+	[ENHANCED_REGULAR_PRESET] = {15, 27, 10},
+	[HIGH_ACCURACY_PRESET] = {47, 83, 20}
+};
+
+static int bmm150_set_power_mode(struct device *dev,
+				enum bmm150_power_modes mode,
```

Comment:
parameters alignment

foo(param1, ...
      paramN, ..., paramK);

That goes for function declaration as well as function calls. Please apply that everywhere (your editor should be able to handle that automatically btw)